tests/TestCase.php

```diff
@@ -97,4 +97,14 @@ public function clean($output)
     {
         return str_replace("\n", "", $output);
     }
+
+    protected function fixDirSlashes(string $path): string
```

Comment:
Can you rename this to `fixDirectorySlashes`, or, if that's too long, something like `fixSlashes` or `fixPath`? We prefer to not use shortened words wherever possible.